client/src/locales/en-US.json

```diff
@@ -103,7 +103,12 @@
         "titles": {
             "workspaces": "My workspaces",
             "settings": "Settings",
-            "devices": "Devices"
+            "devices": "Devices",
+            "activeUsers": "Active users",
+            "revokedUsers": "Revoked users",
+            "invitations": "Invitations",
+            "storage": "Storage",
+            "organization": "Information"
```

Comment:
The key and the content does not seems to match.
Maybe an error or an unclear key name?

Reply:
See my comment for the French translation.